src/ref/lang/auto/ServiceStudio.Plugin.Widgets.EditableTable.md

```diff
@@ -22,7 +22,7 @@ Displays content in a tabular layout and allows users to edit and create records
 <tbody>
 <tr>
 <td title="Name">Name</td>
-<td>Identifies an element in the scope where it is defined, like a screen, action, or module.</td>

```

Comment:
Keep in mind that the edits in some source files (ref/lang/auto/*.md without *begin or *end) will most likely be lost, if not edited in the source code. We'll work on improving instructions on how to handle suggestion for editing those files 👍